app/controllers/idv/address_controller.rb

```diff
@@ -0,0 +1,51 @@
+module Idv
+  class AddressController < ApplicationController
+    include IdvSession
+    include IdvFailureConcern
+
+    attr_reader :idv_form
+
+    before_action :confirm_two_factor_authenticated
+
+    def new
+      analytics.track_event(Analytics::IDV_ADDRESS_VISIT)
+      set_idv_form
```

Comment:
Do you need to call set `set_idv_form` here? Looks like it is not used in the view.